src/menu.ts

```diff
@@ -166,6 +167,17 @@ function getAppMenu( mainWindow: BrowserWindow | null ) {
 					},
 				},
 				{ type: 'separator' },
+				...( process.platform === 'win32'
```

Comment:
I noticed I can still access the prompt from the Help menu, even after previously clicking the "Sounds good, do it for me" prompt. Should it be hidden after the user's already accepted the changes?

Reply:
Good question. I'd hide it if, for any reason, we need to ensure that the process can only be executed once. Since this is not the case, I think we can leave it always visible. Besides, if the user changes the Windows Defender configuration and removes the exclusion, it might be useful to let them update again from the app.